### PR TITLE
move settings to a seperate page in the admin settings

### PR DESCRIPTION
**Pull request by schiessle**
- [x] requires https://github.com/nextcloud/server/pull/796

backport to stable10 as soon as https://github.com/nextcloud/server/pull/796 is backported

### CODE REVIEW

File: lib/Settings/AdminSection.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		$this->l = $l;
+	}
+
+	/**
+	 * returns the ID of the section. It is supposed to be a lower case string
+	 *
+	 * @returns string
+	 */
+	public function getID() {
+		return 'surveyclient';
```
Comment (nickvergessen) [inline lib/Settings/AdminSection.php]: Use the app id? so with `_`

File: lib/Settings/AdminSettings.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		];
+
+		return new TemplateResponse('survey_client', 'admin', $parameters);
+	}
+
+	/**
+	 * @return string the section ID, e.g. 'sharing'
+	 */
+	public function getSection() {
+		return 'surveyclient';
```
Comment (nickvergessen) [inline lib/Settings/AdminSettings.php]: same here

**Comment by schiessle** [comment]
@nickvergessen fixed... :wink:

File: lib/BackgroundJobs/AdminNotification.php [hunk 3]
```diff
@@ -33,7 +33,7 @@ protected function run($argument) {
 			->setDateTime(new \DateTime())
 			->setSubject('updated')
 			->setObject('dummy', 23)
-			->setLink($urlGenerator->getAbsoluteURL('index.php/settings/admin#usage-report'));
+			->setLink($urlGenerator->getAbsoluteURL('index.php/settings/admin/survey_client'));
```
Comment (blizzz) [inline lib/BackgroundJobs/AdminNotification.php]: you can also use linkToRoute() instead, makes you robust against possible future url changes.

**Comment by blizzz** [comment]
One suggestion, works, :+1:

**Comment by blizzz** [comment]
needs rebase

**Pull request merged**